src/css/book.css

```diff
@@ -0,0 +1,142 @@
+
+.card {
+  display: flex;
+  border: 1px solid #ccc;
+  padding: 15px;
+  justify-content: space-between;
+  align-items: center;
+  width: 85%;
+  margin: 1rem auto 0.983rem auto;
+  border-radius: 4px;
+  box-shadow: 0 0 2px 0;
+}
```

Comment:
Great design @danielochuba 🤝

- [ ] Kindly ensure your design replicates the design given as much as possible. Kindly align your elements properly to emulate the [design given](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/images/bookstore.png).
- Your design
![your-cms](https://github.com/danielochuba/bookstore/assets/63917725/25e5c947-4689-4edd-9b65-9594c36532e1)
- I would also suggest you reduce the size of your borders for cards so that your design can replicate the design provided.